src/components/widgets/sparkline/sparkline_dataHelpers.js

```diff
@@ -0,0 +1,88 @@
+
+// import Highcharts from 'highcharts';
+import last from 'lodash/last';
+
+
+export const makeChartOptions = ({
+  emitSetState = () => {},
+  widget,
+}) => {
+  return {
+    // default options
+    chart: {
+      type: 'spline',
+      margin: [150, 0, 0, 0],
+      events: {
+        load: function() {
+          var latestValue = last(this.series[0].data).y
+          this.renderer.text(latestValue)
```

Comment:
I adapted this way of adding extra elements from the examples, I have no idea about the renderer - I guess it must have a renderer available or it wouldn't work :)